tests/conftest.py

```diff
@@ -1,3 +1,19 @@
+from pathlib import Path
+
+import pytest
 from dotenv import load_dotenv
 
+from autogpt.workspace import Workspace
+
 load_dotenv()
+
+
+@pytest.fixture()
+def workspace_root(tmp_path) -> Path:
+    return tmp_path / "home/users/monty/auto_gpt_workspace"
```

Comment:
Make this cross platform by os

Reply:
This will resolve to a windows path if run on windows.  pathlib obfuscates os specific pathing.